dev/io.openliberty.springboot.fat30_fat/fat/src/com/ibm/ws/springboot/support/fat/CommonWebFluxTests.java

```diff
@@ -72,9 +72,14 @@ public void testBlockingIO() throws IOException, InterruptedException {
         con.connect();
 
         OutputStream out = con.getOutputStream();
-        for (int i = 0; i < numIterations; i++) {
-            Thread.sleep(sleep);
-            out.write(data);
+        try {
+            for (int i = 0; i < numIterations; i++) {
+                Thread.sleep(sleep);
+                out.write(data);
+            }
+        } finally {
+            out.flush();
+            out.close();
```

Comment:
It is unclear to me that we should close the OutputStream on the `HttpURLConnection` while it is still alive.  But I think we can give it a try to see if it solves the intermittent issue.  Alternatively, I would have suggested only to do a `flush` call here.